src/oc/web/actions.cljs

```diff
@@ -1002,4 +1006,82 @@
                                 (assoc :reactions-loading next-reactions-loading)
                                 (assoc-in [:reactions reaction-idx] next-reaction-data))
             updated-entries-data (assoc entries-data entry-idx updated-entry-data)]
-        (assoc-in db topic-entries-key updated-entries-data)))))
\ No newline at end of file
+        (assoc-in db topic-entries-key updated-entries-data)))))
```

Comment:
nope, `topic-entries-key` is a vector of keywords https://github.com/open-company/open-company-web/blob/dynamic/src/oc/web/dispatcher.cljs#L50-L51